katsdpcontroller/scheduler.py

```diff
@@ -1219,6 +1219,14 @@ def __str__(self):
                 "to launch all tasks ({0.needed} > {0.available})".format(self))
 
 
+class QueueBusyError(InsufficientResourcesError):
+    """The launch group did not reach the front of the queue before its timeout expired."""
+    def __init__(self):
+        super().__init__(
+            "The launch group did not reach the front of the queue before its timeout expired"
```

Comment:
Please include the timeout itself just to aid debugging.

Reply:
Done.